### ISSUE TITLE
[A11y 8/25]  focus with keyboard navigation

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-menu-behavior#22]
When I use the cursor to click on an item, there is a blue focus state showing that it is highlighted. This doesn't occur when I am just using the keyboard. I should be able to use the arrow keys to navigate through and press enter to get the same result.

**Comment by blasten** [comment]
The blue background is for the selected state. Once https://github.com/PolymerElements/iron-selector/pull/58 is merged, users will be able to select items using the keyboard (enter key).

**Comment by blasten** [comment]
This should be fixed now after merging https://github.com/PolymerElements/iron-selector/pull/36
